### DISCUSSION TITLE
Extremely slow download on Samsung A32 4G

### DISCUSSION THREAD

**Discussion by AYakovleff** [TeamNewPipe/NewPipe#10363] [Questions]
Hello everyone.  NP SB 0.25.2 is being used on a Samsung A32 SM-A325F/DS Android 13 and it downloads videos terribly slow regardless what source it is: YT or Soundcloud or whatever. Restart, cache wipe does not help. Network speed indicator shows down speed of 0-1 kbps only. Also used it before in spring 2023 when Android 11 was installed, probably version 0.25.0 or 0.25.1 (993) and NPSB worked great. 
May the reason be in the 13-th version of Android or Samsung specific protections they've embedded? Don't you know how to fix it?
Any answer would be highly appreciated.

**Comment by opusforlife2** [comment]
https://github.com/TeamNewPipe/NewPipe/issues/new/choose

**Reply by AYakovleff** [reply]
Sorry, should I issue a new bug report or?

**Reply by opusforlife2** [reply]
Yes. And please follow the instructions in the template.
